include/grpc++/server_credentials.h

```diff
@@ -54,6 +55,10 @@ class ServerCredentials {
 
   virtual int AddPortToServer(const grpc::string& addr,
                               grpc_server* server) = 0;
+
+  // Has to be called before the server is started.
```

Comment:
Maybe add a comment saying should only be called once or the last one counts or something?